systers_portal/systers_portal/settings/base.py

```diff
@@ -164,3 +164,14 @@
 
 # Django Crispy Forms configuration
 CRISPY_TEMPLATE_PACK = 'bootstrap3'
+
+# Settings for Facebook OAuth
+SOCIALACCOUNT_PROVIDERS = \
+    {'facebook': {'METHOD': 'oauth2',
+                  'SCOPE': ['email'],
+                  'AUTH_PARAMS': {'auth_type': 'reauthenticate'},
+                  'FIELDS': ['id', 'email'],
+                  'EXCHANGE_TOKEN': True,
+                  'LOCALE_FUNC': lambda request: 'kr_KR',
```

Comment:
This would return the Korean JS SDK :)

Reply:
@SanketDG Actually its from a medium article. @ritwickraj78 Please change it :(

Reply:
I am absolutely aware, I just did not want to point it out. It's totally fine, it was probably a mistake, no worries at all.

Reply:
Agreed. You can always take help @ritwickraj78

Reply:
@sakshi1499 @SanketDG It is the same blog whose instructions I had sent May. Mistakenly I copied from there rather than the official docs. And yes absolutely my fault. Won't happen again.
I am making the changes. Please review it.